app/models/podcast.rb

```diff
@@ -32,7 +32,9 @@ def existing_episode(item)
     episode = PodcastEpisode.where(media_url: item.enclosure_url).
       or(PodcastEpisode.where(title: item.title)).
       or(PodcastEpisode.where(guid: item.guid.to_s)).presence
-    episode ||= PodcastEpisode.where(website_url: item.link).presence if unique_website_url?
+    # if unique_website_url? is set to true (the default value), we try to find an episode by website_url as well
+    # if unique_website_url? is set to false it usually means that website_url is the same for different episodes
+    episode ||= PodcastEpisode.where(website_url: item.link).presence if item.link.present? && unique_website_url?
```

Comment:
Currently, `unique_website_url?` can only be set from the console, and is set to `false` only for one of the podcasts ("Base cs podcast"). I don't like the fact that this logic is hidden from the community admins but at the same time it is used, so we can't just remove it.

Reply:
How would we get rid of it?

Reply:
The "feature" is not explicitly used, and, for now, I don't know if there are any episodes actually found by `website_url`, which are not possible to find by other fields.
So, it would be nice to explore this and either remove the feature or make it explicit.

Reply:
Let's add a card somewhere in our projects columns, we can explore this at some point. So we don't forget